packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -772,7 +784,11 @@ class DropdownCore extends React.Component<Props, State> {
                     styles.dropdown,
                     light && styles.light,
                     isReferenceHidden && styles.hidden,
-                    {minWidth: minDropdownWidth},
```

Comment:
I'm not sure if I found an issue with how we merge styles in `wonder-blocks-core`.

Turns out that in this scenario:

```
<SingleSelect dropdownStyle={{ minWidth: 100 }} ...>
```

`minWidth=100` was overridden by `DropdownCore` internally because using the object in line #775 takes more precedence because WB core takes this as an inline style so it appends it at the last position of the array styles.

I'm not sure if this is expected or if it's some issue that we haven't noticed yet.